release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -148,6 +148,90 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-vpshere-top-support-multi-datacenters"]
+==== VMware vSphere CSI topology support for multiple datacenters is generally available
+{product-title} 4.12 introduced the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters, thus helping to avoid a single point of failure. {product-title} 4.13 introduces support for deploying over multiple datacenters also.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[vSphere CSI topology].
+
+[id="ocp-4-13-storage-multiple-default-sc"]
+==== Creating more than one default storage class is generally available
+{product-title} 4.13 allows you create more than one default storage class. This feature makes it easier to change the default storage class because you can create a second storage class defined as the default. You then temporarily have two default storage classes before removing default status from the previous default storage class. While it is acceptable to have multiple default storage classes for a short time, you should ensure that eventually only one default storage class exists.
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-managing-default-sc"]
+==== Managing the default storage class is generally available 
+{product-title} 4.13 introduces the `spec.storageClassState` field in the `ClusterCSIDriver` object, which allows you to manage the default storage class to accomplish several different objectives:
+
+* Enforcing static provisioning by disabling dynamic provisioning.
+
+* When you have other preferred storage classes, preventing the storage operator from re-creating the initial default storage class.
+
+* Renaming, or otherwise changing, the default storage class
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-retroactive-default-sc-assignment"]
+==== Retroactive default StorageClass assignment (Technology Preview)
+There are some cases where the default storage classes are not assigned, or more than one default is defined. This feature allows you to retroactively assign a default storage class to correct these situations. This feature is supported with Technology Preview status.
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-csi-inline-ephemeral-vols-GA"]
+==== CSI inline ephemeral volumes is generally available
+Container Storage Interface (CSI) inline ephemeral volumes were introduced in {product-title} 4.5 as a Technology Preview feature, which allows you to define a pod spec that creates inline ephemeral volumes when a pod is deployed and delete them when a pod is destroyed. This feature is now generally available.
+
+This feature is only available with supported Container Storage Interface (CSI) drivers.
```

Comment:
We'd better also metion that only when csi driver is not support `Generic ephemeral volumes`, it'll support CSI inline ephemeral and we recommend customers to use the `Generic ephemeral volumes`. WDYT?

Reply:
AFAIK all drivers that supports dynamic provisioning supports generic ephemeral volume.

Reply:
Yeah, all the playload dynamic provisioning csi driver support generic ephemeral volume, but looks like only few csi driver support the CSI inline ephemeral volumes(E.g. sharedresource csi driver and hostpath csi driver ), so maybe we could make the CSI inline ephemeral volumes support matrix clearly.

Reply:
right, I haven't reviewed the CSI inline doc yet, if we want some kind of matrix or listing that would be there not in the release notes.

Reply:
right, I haven't reviewed the CSI inline doc yet, if we want some kind of matrix or listing that would be there not in the release notes.

Reply:
Got it. Thanks for clarifying!

Reply:
@Phaow @gcharot do we want this covered in the main docs? Agreed it doesn't belong in RNs.

Reply:
IMO, this could go into https://github.com/openshift/openshift-docs/pull/47603, there are only 2 drivers that ship with OCP and support inline volumes.